sample_configs/aws-cloudtrail/README.md

```diff
@@ -0,0 +1,71 @@
+## CloudTrail to Firehose-Elasticsearch
+
+### Description
+
+Writes CloudTrail logs stored in S3 to Firehose steam which pushes to an AWS hosted Elasticsearch cluster. Consult [es services docs](https://aws.amazon.com/elasticsearch-service/) on how to setup Elasticsearch service and [CloudTrail docs](https://docs.aws.amazon.com/console/awscloudtrail/) on how to setup CloudTrail.
```

Comment:
Line-wrap @ 80chars everywhere in Markdown pls.. (unless absolutely not possible)